gh-102812: unittest's addError/addFailure/addSubTest accept an exception instance in addition to (typ, val, tb) as paramter (#102814)
python/cpython: Doc/whatsnew/3.12.rst

  same name on a base class, as per :pep:`698`. (Contributed by Steven Troxler in
  :gh:`101564`.)

unittest
--------

:meth:`~unittest.TestResult.addError`,
:meth:`~unittest.TestResult.addFailure` and

> :meth:`~unittest.TestResult.addSubTest` now accept either an exception
> instance or a ``(typ, val, tb)`` tuple. (Contributed by Irit Katriel in

Reviewer: Make clear what is new while being clear that still only 1 is passed.
```suggestion
:meth:`~unittest.TestResult.addSubTest` now accept an exception instead of
a ``(typ, val, tb)`` tuple. (Contributed by Irit Katriel in
```
Do you want to say that an exception is in some sense preferred?

Author: I think "instead of" may sound like the old format is no longer accepted. But agree I would emphasise what is new and what is old.
